src/app/Shared/LoadingOrError.component.tsx

```diff
@@ -11,16 +11,15 @@ import { LoadingProcessComponent } from "./LoadingProcess.component";
 export function LoadingOrErrorComponent(props: {
     status:
         | {
-              isolateStatus: number;
+              isolateStatus: number | undefined;
               isolateCountStatus: number | undefined;
-              filterStatus: number;
+              filterStatus: number | undefined;
           }
-        | undefined;
     dataIsSet: boolean;
     componentToDisplay: JSX.Element;
 }): JSX.Element {
     if (props.status !== undefined) {
-        if (props.status.isolateStatus !== 200) {
+        if (props.status.isolateStatus !== 200 && props.status.isolateStatus) {

```

Comment:
reject (also below): check explicitly against undefined

Reply:
Done. Commit "Change if-conditions in LoadingOrError"